tests/topotests/bgp_conditional_advertisement/test_bgp_conditional_advertisement.py

```diff
@@ -0,0 +1,961 @@
+#!/usr/bin/env python
+
+#
+# test_bgp_conditional_advertisement.py
+# Part of NetDEF Topology Tests
+#
+# Copyright (c) 2020 by
+# Network Device Education Foundation, Inc. ("NetDEF")
```

Comment:
Wrong copyright?

Reply:
> 
> 
> Wrong copyright?

@rzalamena I thought it should not be changed. will change it. Thanks for reviewing and approving the code changes